api/application/client.go

```diff
@@ -772,3 +772,33 @@ func (c *Client) UnsetApplicationConfig(application string, options []string) er
 	}
 	return results.OneError()
 }
+
+// ResolveUnitErrors clears errors on one or more units.
+// Either specify one or more units, or all.
+func (c *Client) ResolveUnitErrors(units []string, all, retry bool) error {
+	// convert unit name to a tag
+	if len(units) > 0 && all == true {
+		return errors.New("provide units or '--all' but not both")
+	}
+	args := params.UnitsResolved{
+		All:   all,
+		Retry: retry,
+	}
+	if !all {
+		entities := make([]params.Entity, len(units))
```

Comment:
Should we be checking for duplicates? The same unit specified twice?

Reply:
a check won't hurt, done